koku/api/settings/tags/mapping/view.py

```diff
@@ -35,6 +34,15 @@
 from reporting.provider.all.models import TagMapping
 
 
+def parse_list(string_to_parse):
+    """Parse a list to a string."""
+    if "[" not in string_to_parse:
+        return [string_to_parse]
+    split_values = string_to_parse.strip("[]").split(",")
+    parse_list = [value.strip().strip("'\"") for value in split_values]
```

Comment:
Also strip leading/trailing spaces.
```suggestion
    parse_list = [value.strip().strip("'\" ") for value in split_values]
```